### DISCUSSION TITLE
Disabling the capability to create org/repo level secret

### DISCUSSION THREAD

**Discussion by lmilbaum** [community/community#49619] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am not sure the capability is available. 
As a platform engineer, I'd like to track the usage of GitHub secrets, disable/activate the capability and to generate a report or a usage chart.

**Comment by lmilbaum** [comment]
The discussion thread is no longer visible which prevents me from providing my feedback to the last reply. Is it a bug or a feature?

**Reply by guy9050** [reply]
Seems that whoever opened the thread had their account removed. If you see the "Events" tab on the left side if this page, it says  "Hi, I'm @ghost! I take the place of user accounts that have been deleted. ghost".

Probably the discussion was removed because the user was removed.

**Reply by lmilbaum** [reply]
Ohh. Totally makes sense. Thanks @guy9050 for the clarification.
